use-cases/index.rst

```diff
@@ -37,10 +36,9 @@ Computer Vision for Medical Imaging
 .. toctree::
     :maxdepth: 1
 
-    computer_vision/1-metastases-detection-train-model
-    computer_vision/2-metastases-detection-lineage-registry
-    computer_vision/3-metastases-detection-deploy-predict
-    computer_vision/4-metastases-detection-pipeline
+    computer_vision/metastases-detection-pipeline.ipynb
+    use-cases/computer_vision/metastases-detection.ipynb
```

Comment:
There are 2 notebooks listed under the `Computer Vision for Medical Imaging` section, but on the RTD preview, I only see 1 link under this section (`Computer Vision for Medical Imaging - Pipeline Mode`) - should there be a second link generated?

Reply:
Oh whoops, looks like I mistyped the link, let me correct that